inc/V4l2Device.h

```diff
@@ -18,8 +18,6 @@
 #include <linux/videodev2.h>
 #include <fcntl.h>
 
-#define V4L2WRAPPER_SUPPORT_PARTIAL_WRITE   1
```

Comment:
This reminds me, we need an API for programs to query whether the v4l2wrapper supports partial write or not. We need a function or library version number.